tc39/spec.html

```diff
@@ -1573,6 +1573,18 @@
 <emu-clause id="simd-proto">
 <h1>The _SIMD_Constructor.prototype</h1>
 
+<emu-clause id="this-simd-value" aoid="thisSIMDValue">
```

Comment:
Perhaps useful to add something like this here ? Similar to Number prototype.
>Unless explicitly stated otherwise, the methods of the SIMD prototype object defined below are not generic and the this value passed to them must be either a SIMD value or an object that has a [[SIMDData]] internal slot that has been initialized to a SIMD value.